GTViewController/Classes/Custom Classes/GTFollowupModalView.m

```diff
@@ -70,16 +70,12 @@ - (instancetype)initFromElement:(TBXMLElement *)element withStyle:(GTPageStyle*)
         modalComponentElement = modalComponentElement->nextSibling;
     }
     
-    // there are too many elements, not sure what to do
-    if (totalUsedSpace >= totalVerticalSpace) {
-        
-    } else {
-        CGFloat availableBufferSpace = totalVerticalSpace - totalUsedSpace;
-        topLeadingSpace =       availableBufferSpace * 0.15;
-        betweenElementsSpace =  (availableBufferSpace * 0.2) / (numLabels + numButtonPairs + numTextInputs);
-    }
+    CGFloat availableBufferSpace = totalVerticalSpace - totalUsedSpace;
+    topLeadingSpace =       availableBufferSpace * 0.15;
+    betweenElementsSpace =  (availableBufferSpace * 0.2) / (numLabels + numButtonPairs + numTextInputs);
     
     int currentY = topLeadingSpace;
+    int inputFieldTag = 0;
```

Comment:
I'm not sure if tag conflicts affect anything in the sdk but they are bad in our code because our animation logic depends on them not conflicting.